src/components/SingleTextArea.tsx

```diff
@@ -0,0 +1,20 @@
+import { FormControl, FormLabel, Select, Textarea } from '@chakra-ui/react';
+import React from 'react';
+
+export default function SingleTextArea({
+  text,
+  valueProp,
+  handleOnChange,
+}: {
+  text: string;
+  valueProp: string;
+  handleOnChange: (e: React.ChangeEventHandler<HTMLTextAreaElement>) => void;
+}) {
+  return (
+    <FormControl>
+      <FormLabel>{text}</FormLabel>
+      {/* ts + onChange, jak to ogarnac? */}
```

Comment:
a jaki jest problem/błąd?

Reply:
ts: (nie wiem jak to otypowac)

Type '(e: ChangeEventHandler<HTMLTextAreaElement>) => void' is not assignable to type 'ChangeEventHandler<HTMLTextAreaElement>'.
  Types of parameters 'e' and 'event' are incompatible.
    Type 'ChangeEvent<HTMLTextAreaElement>' is not assignable to type 'ChangeEventHandler<HTMLTextAreaElement>'.
      Type 'ChangeEvent<HTMLTextAreaElement>' provides no match for the signature '(event: ChangeEvent<HTMLTextAreaElement>): void'.ts(2322)

Reply:
handleOnChange: ChangeEventHandler<HTMLTextAreaElement>;
funkcja podawana:
 const handleTextAreaChange = (e: ChangeEvent<HTMLTextAreaElement>)